apps/server/src/adapterSetup.ts

```diff
@@ -1,25 +1,28 @@
 import { getMxAdapterMapObject } from "@ucp-npm/mx-adapter";
+import { getSophtronAdapterMapObject } from "@ucp-npm/sophtron-adapter";
 import type { AdapterMap } from "@repo/utils";
 
 import config from "./config";
 import { get, set } from "./services/storageClient/redis";
 import * as logger from "./infra/logger";
-import { SophtronAdapter } from "./adapters/sophtron";
 
-import getSophtronVc, {
-  dataAdapter as sophtronDataAdapter,
-} from "./services/vcAggregators/sophtronVc";
 import { adapterMapObject as testAdapterMapObject } from "./test-adapter";
 
-const sophtronAdapterMapObject: Record<string, AdapterMap> = {
-  sophtron: {
-    dataAdapter: sophtronDataAdapter,
-    vcAdapter: getSophtronVc,
-    widgetAdapter: new SophtronAdapter(),
-  },
-};
+const sophtronAdapterMapObject: Record<string, AdapterMap> =
+  getSophtronAdapterMapObject({
+    logClient: logger,
+    aggregatorCredentials: {
+      clientId: config.SophtronApiUserId,
+      secret: config.SophtronApiUserSecret,
+      endpoint: config.SophtronApiServiceEndpoint,
```

Comment:
should endpoint and vcEndpoint just be hardcoded into the adapter?

Reply:
Good catch.

Reply:
We should do the same for the MX adapter.